src/runtime/minrpc/minrpc_intrfc.h

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Could you spell out the full name, i.e. make the file name `minrpc_interface.h`?

Reply:
done